### ISSUE TITLE
Question, is there a way to place the dropped file in the dotted area after uploaded?

### ISSUE THREAD

**Issue by tkiethanom** [paramaggarwal/react-dropzone#66]
I tried just putting the generated markup in the Dropzone tag but it complained about the DOM being mutated.

```javascript
<Dropzone onDrop={(files) => this.handleFileUpload(files) } multiple={false}>
	<div>Try dropping some a file here, or click to select a file to upload.</div>
	{filesMarkup}
</Dropzone>
```

**Comment by paramaggarwal** [comment]
Please try the example where it uses `file.preview`.

**Comment by tkiethanom** [comment]
Yes, I am using `file.preview`. In my example `{filesMarkup}` creates the <img> tag with `file.preview` as the src. This works fine when it is outside of the <Dropzone> tags. However when I put it inside the <Dropzone> tag it complains the DOM has been mutated. 

```javascript
render(){
    let filesMarkup;
    if (_.isEmpty(this.props.App.files) === false ) {
	    filesMarkup = this.props.App.files.map((file) =>
		  <img src={file.preview} style={{width: '100px'}}/>
            );
    } else {
	    filesMarkup = '';
    }

    return (
        <Dropzone onDrop={(files) => this.handleFileUpload(files) } multiple={false}>
            <div>Try dropping some a file here, or click to select a file to upload.</div>
            {filesMarkup}
        </Dropzone>
    );
}

handleFileUpload(files){
	const { dispatch } = this.props;
	dispatch(fileUploaded(files));
}
```

**Comment by tkiethanom** [comment]
When I inspect the React elements after an upload it looks like its trying to put an `<img>` tag inside of an `<input>` which is probably the problem. Which makes sense since `<Dropzone>` is actually an input. 

Here is the error:

*Uncaught Error: Invariant Violation: processUpdates(): Unable to find child 1 of element. This probably means the DOM was unexpectedly mutated (e.g., by the browser), usually due to forgetting a <tbody> when using tables, nesting tags like <form>, <p>, or <a>, or using non-SVG elements in an <svg> parent. Try inspecting the child nodes of the element with React ID `.0.0.1.0.0.$=14.0`.*

**Comment by paramaggarwal** [comment]
Ah I see. This is fixed in the latest `v2.1.0`.
